src/domain/Users/services/getProjectsByUserId.service.test.ts

```diff
@@ -0,0 +1,27 @@
+import "reflect-metadata";
+import { container } from "tsyringe";
+import { PROJECTS_LIST_MOCK } from "../repositories/repositoryMock";
+import { type GetProjectsByUserIdService } from "./getProjectsByUserId.service";
+
+describe("Get Projects By User ID", () => {
```

Comment:
Could you please change the describe title to something with a patter? For example
```suggestion
describe("service: GetProjectsByUserIdService", () => {
```

Reply:
Yes i can, thank you